src/retasc/models/prerequisites/jira_issue.py

```diff
@@ -75,7 +75,7 @@ def _edit_issue(
 
 def _report_jira_issue(issue: dict, jira_issue_id: str, context):
     context.report.jira_issues[jira_issue_id] = issue
-    context.report.set("issue", issue["key"])
+    context.report.set("issue_id", issue["key"])
```

Comment:
```suggestion
    context.report.set("jira_issue", issue["key"])
```
jira_issue?
the jira_issue_id above more sound fit here, but it seems a issue object?
Below you have jira_issue that get set with jira_issue_id, bit confusing the names and their purposes

Reply:
This is Jira issue ID like `RHELW-123`. `jira_issue_id` contains the human-readable ID used to construct the label that is used to uniquely identify the issue in Jira.